components/locale_canonicalizer/src/locale_canonicalizer.rs

```diff
@@ -17,9 +17,25 @@ pub enum CanonicalizationResult {
     Unmodified,
 }
 
+/// A bag of options controlling how the LocaleCanonicalizer is initialized.
+#[derive(Debug, Eq, PartialEq, Clone, Default)]
+pub struct LocaleCanonicalizerOptions {
+    /// Controls whether to load alias data to support the canonicalize operation.
+    load_canonicalization_data: bool,
```

Comment:
`suggestion (non-blocking)` - I would name it `aliases` or something similar rather than `load_x`.

Reply:
As discussed, we'll make this a feature controlled at build time.